setup.py

```diff
@@ -6,11 +6,28 @@
 """
 from setuptools import setup, find_packages
 
+
 # get the version
 version = {}
 with open('bigchaindb/version.py') as fp:
     exec(fp.read(), version)
 
+
+# check if setuptools is up to date
+def check_setuptools_features():
+    import pkg_resources
+    try:
+        list(pkg_resources.parse_requirements('foo~=1.0'))
+    except pkg_resources.RequirementParseError:
```

Comment:
This creates another problem because with older versions of setuptools `pkg_resources.RequirementsParseError` may not exists. I think this is the case with ubuntu 14.04 which from the repos installs setuptools v3.0.0